cloudbuild.yaml

```diff
@@ -70,7 +70,14 @@ steps:
       - |
         gcloud secrets versions access latest --secret=$_GOOGLE_SERVICES_JSON --format='get(payload.data)' | tr '_-' '/+' | base64 -d > ground/google-services.json
 
-  - name: 'gcr.io/$PROJECT_ID/android:30'
+  - name: 'gcr.io/$PROJECT_ID/android:34'
+    id: &print_debug 'Print debug info'
+    args:
+      - '-c'
+      - |
+        ./gradlew --version
+
```

Comment:
Can we cleanup this if it is not needed anymore?

Reply:
I think it's still useful to have for understanding which version of things are being used to build. For ex:

<img width="535" alt="image" src="https://github.com/google/ground-android/assets/228050/064ce8fd-747e-44cf-b2c6-a9a8fe62dc7a">

Wdyt?